.github/workflows/enact-migration.yaml

```diff
@@ -55,15 +55,19 @@ jobs:
           esac
 
       - name: Seacrest
-        uses: hayesgm/seacrest@v1
+        uses: hayesgm/seacrest@c3eef919c8ac21cf077aad0513ad8a493885ce6d
         with:
+          wallet_connect_project_id: ${{ secrets.WALLET_CONNECT_PROJECT_ID }}
+          requested_network: "${{ fromJSON('{\"fuji\":43113,\"mainnet\":1,\"goerli\":5,\"mumbai\":80001,\"polygon\":137,\"arbitrum-goerli\":421613,\"arbitrum\":42161,\"base-goerli\":84531,\"linea-goerli\":59140}')[inputs.network] }}"
           ethereum_url: "${{ fromJSON('{\"fuji\":\"https://api.avax-test.network/ext/bc/C/rpc\",\"mainnet\":\"https://mainnet.infura.io/v3/$INFURA_KEY\",\"goerli\":\"https://goerli.infura.io/v3/$INFURA_KEY\",\"mumbai\":\"https://polygon-mumbai.infura.io/v3/$INFURA_KEY\",\"polygon\":\"https://polygon-mainnet.infura.io/v3/$INFURA_KEY\",\"arbitrum-goerli\":\"https://arbitrum-goerli.infura.io/v3/$INFURA_KEY\",\"arbitrum\":\"https://arbitrum-mainnet.infura.io/v3/$INFURA_KEY\",\"base-goerli\":\"https://base-goerli.infura.io/v3/$INFURA_KEY\",\"linea-goerli\":\"https://linea-goerli.infura.io/v3/$INFURA_KEY\"}')[inputs.network] }}"
           port: 8585
         if: github.event.inputs.eth_pk == ''
 
       - name: Seacrest (governance network)
-        uses: hayesgm/seacrest@v1
+        uses: hayesgm/seacrest@c3eef919c8ac21cf077aad0513ad8a493885ce6d
         with:
+          wallet_connect_project_id: ${{ secrets.WALLET_CONNECT_PROJECT_ID }}
+          requested_network: "${{ fromJSON('{\"fuji\":43113,\"mainnet\":1,\"goerli\":5,\"mumbai\":80001,\"polygon\":137,\"arbitrum-goerli\":421613,\"arbitrum\":42161,\"base-goerli\":84531,\"linea-goerli\":59140}')[inputs.network] }}"
```

Comment:
`inputs.network` should be `env.GOV_NETWORK`

Reply:
ok 👍